src/lib/dropdowns/language_dropdown/toggle.js

```diff
@@ -0,0 +1,70 @@
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
+import { bool, func, string } from 'prop-types';
+import { color, space } from '@core-ds/primitives';
+
+const ToggleElem = styled.input`
+   appearance: none;
+   position: relative;
+   width: 1.75rem;
+   height: 1rem;
```

Comment:
IIRC we shouldn't use rem because the Shopify theme sets the base size differently than ours. Might not matter since the language dropdown doesn't make sense on Shopify anyway.